proxies/shared/resources/jsc/Messages.Create.Request.js

```diff
@@ -1,10 +1,26 @@
 const content = context.getVariable("request.content")
 const data = JSON.parse(content).data
+var message = null;
```

Comment:
These changes don't look correct, the proxy should be passing the POST'd content through to the backend - just because the POST /api/v1/send endpoint has a badly designed request/response model doesn't mean this new endpoint should.

We've raised a ticket to address fixing the legacy of not being able to change that endpoint due to go-live of the service - CCM-1924

Reply:
okay, ive changed it around